### ISSUE TITLE
Source list refuses to load after downloading keiyoushi extension

### ISSUE THREAD

**Issue by DeadassJT** [mihonapp/mihon#457]
### Steps to reproduce

1. Open keioushi.github.io/extensions/
2. Tap on ‘Add repository’ button
3. Exit Mihon app
4. Clear app history
5. Open Mihon again and go to browse>extensions
6. Refresh extensions
7. Page stays refreshing and sources do not appear

### Expected behavior

The source list should appear once refreshing is done.

### Actual behavior

Loading icon stays on top of empty page and continues to load for several seconds and then dissapears with no change to the page.
![76C90CCC-E761-4B68-BB28-16EF44C8014C](https://github.com/mihonapp/mihon/assets/161398568/23f0bb19-5776-4f3b-96b0-0b10a392daa7)

### Mihon version

0.16.4

### Android version

Android 10

### Device

Motorola model g7 play

### Other details

I am currently trying to migrate from tachiyomi to Mihon and at first, I possessed all my old sources when trying to use the keiyoshi repo. After checking everywhere I could, I had to open the discord before I saw any information stating I was supposed to uninstall all old sources. So once I did I went to settings and cleared the storage of the Mihon app before starting again (minus restoration of backup this time) and even then, nothing refuses to load.

I am also not 100% sure whether I was supposed to post this issue on the keiyoushi page or here, if I'm posting in the wrong place, please inform me.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.4](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.
